_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,54 @@
+
+The researchers modeled what heat signals might come off an exoplanet, using Earth as a basis.
+Weighing the energy inputs (solar and planetary heat sources) and outputs (photosynthesis and other biological activity as well as technology) for their model planet, they reached the conclusion that as you produce more energy that is returned to the planet as heat, the model planet starts to warm up.
+As the authors put it, "a civilization with growing power needs will eventually reach a point where they become uncomfortably warm."
+
+Even if the alien civilizations found a way to engineer their planet to reflect heat out of the atmosphere, this would still leave a detectable heat signature.
+The more advanced a civilization is, the argument goes, the more energy it needs, and the more it will move towards harnessing all the available energy in its solar system, perhaps culminating in a [Dyson sphere](http://en.wikipedia.org/wiki/Dyson_sphere).
+We can compare energy consumption with the theoretical maximum available (i.e. the flux of the host star) to get a measure of how advanced (in terms of energy consumption and thus heat output) a civilization is.
+For Earth, the researchers estimate we are now consuming 0.04% of the flux of the Sun; more advanced civilizations will tend towards 100% (the Dyson sphere scenario).
+
+For their computer model, the researchers gave their simulated exoplanet Earth-like properties of reflectance, scattering, and geography.
+The brightness of the planet changed predictably with its programmed rotation and planetary orbit period.
+By subtracting out short-wavelength light, which is the most sensitive marker of the planet's albedo or percentage of reflected radiation, they arrived at a measure of the planetary flux for a simulated civilization with a thermal heat signal 50 times larger than Earth's.
+
+Looking at the visible and infrared light coming off a planet, it is thus in principle possible to ascertain thermal signatures compatible with alien life.
+This method could be enhanced by coupling it with existing tools like spectroscopy, which looks at the reflected spectrum of electromagnetic radiation to determine the chemical composition of exoplanets and stars.
+Still, the heat signature will be confounded by planets completely covered in clouds or water.
+
+In order to find these hot planets, really big telescopes are needed, the authors argue.
+The challenge is to achieve sufficient contrast sensitivity &mdash; being able to distinguish the planet's faint light from the brightness of the star it orbits &mdash; and for that you need telescopes with [large-diameter objectives](http://en.wikipedia.org/wiki/Objective_%28optics%29#Telescope), plus other technology to block light from other stars and focus the light you _do_ want to see.
```

Comment:
"the planet's faint thermal signatures from the heat generated by the star it orbits"?